meshtastic/node.py

```
@@ -105,27 +197,63 @@ def showInfo(self):
-        """Send regular MeshPackets to ask channels."""
+    def requestChannels(self, startingIndex: int = 0) -> None:
+        """
+        Request channel definitions from the node, starting at the given channel index.
+
+        When called with startingIndex 0, clears any cached channels and begins a fresh fetch into
+        an internal partialChannels list. The method initiates the network request for the
+        specified channel index.
+
+        Parameters
+        ----------
+            startingIndex (int): Zero-based channel index to start fetching from (typically 0–7).
```

Ruff (0.15.1)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)